src/libsyntax_ext/format.rs

```diff
@@ -543,6 +543,10 @@ impl<'a, 'b> Context<'a, 'b> {
         let mut pats = Vec::new();
         let mut heads = Vec::new();
 
+        let names_pos: Vec<_> = (0..self.args.len()).map(|i| {
+            self.ecx.ident_of(&format!("arg{}", i)).gensym()
```

Comment:
Bikeshed opportunity: `arg0` or `arg_0`?

Reply:
_shrug_ this is never exposed so it doesn't matter